src/views/TableDetail.vue

```diff
@@ -18,7 +18,7 @@
 
           <v-divider />
 
-          <div v-if="loading">
+          <div v-if="loadingTables">
```

Comment:
Is `loading` no longer used in this component anywhere?

Reply:
It's used for the rows in the table when the table changes. The problem was that when you were in the view and clicked on another table, you'd have the skeleton loaders pop back in for the table list, even though the list of tables was loaded and still visible under the skeletons.